demos/paint_your_dreams_demo/main.py

```diff
@@ -196,15 +214,22 @@ def build_ui() -> gr.Interface:
                             stop_button = gr.Button("Stop generation", variant="secondary")
 
             with gr.Accordion("Advanced options", open=False):
+                #global hf_model_name
+                if hf_model_name == "dreamlike_anime_1_0_fp16_ov":
+                    guidance_scale_value = 7.5
+                    num_inference_steps = 50
+                else:
+                   guidance_scale_value = 8.5
+                   num_inference_steps = 4
```

Comment:
The configuration should be provided as a dictionary at the top of the script. It must include all supported models e.g. LCM